t/features/builtins/methods.t

```diff
@@ -309,5 +309,47 @@ use _007::Test;
     is-result $ast, "True\nFalse\n", "String.contains() works";
 }
 
+{
+    my $program = q:to/./;
```

Comment:
That's just a git add missing, sorry.

My workflow is skewed because I git add -p often without git status...